backend/src/ee/services/dynamic-secret/dynamic-secret-service.ts

```diff
@@ -144,13 +145,14 @@ export const dynamicSecretServiceFactory = ({
 
     const projectId = project.id;
 
-    const { permission } = await permissionService.getProjectPermission(
+    const { permission, ForbidOnInvalidProjectType } = await permissionService.getProjectPermission(
       actor,
       actorId,
       projectId,
       actorAuthMethod,
       actorOrgId
     );
+    ForbidOnInvalidProjectType(ProjectType.SecretManager);
```

Comment:
comment: wow so, we now always have to define this? or is this just for old services and new ones will be scoped already ?

Reply:
moved  to seperate pr